src/metabase/driver/sparksql_databricks_v2.clj

```diff
@@ -203,20 +213,26 @@
         (throw e)))))
 
 ;; the current HiveConnection doesn't support .createStatement
-(defmethod sql-jdbc.execute/statement-supported? :sparksql-databricks [_] false)
-
-(doseq [feature [:basic-aggregations
-                 :binning
-                 :expression-aggregations
-                 :expressions
-                 :native-parameters
-                 :nested-queries
-                 :standard-deviation-aggregations]]
-  (defmethod driver/supports? [:sparksql-databricks feature] [_ _] true))
+(defmethod sql-jdbc.execute/statement-supported? :sparksql-databricks-v2 [_] false)
+
+(doseq [[feature supported?] {:basic-aggregations               true
+                              :binning                          true
+                              :expression-aggregations          true
+                              :expressions                      true
+                              :native-parameters                true
+                              :nested-queries                   true
+                              :standard-deviation-aggregations  true
+                              :foreign-keys                     true
+                              :full-join                        true
+                              :right-join                       true
+                              :left-join                        true
+                              :inner-join                       true
+                              :window-functions/offset          true}]
+  (defmethod driver/database-supports? [:sparksql-databricks-v2 feature] [_driver _feature _db] true))
```

Comment:
```suggestion
  (defmethod driver/database-supports? [:sparksql-databricks-v2 feature] [_driver _feature _db] supported?))
```
Allows to disable features by putting some to false above instead of ignoring values